lib/pure/times.nim

```diff
@@ -522,6 +523,35 @@ proc getDaysInYear*(year: int): int =
     doAssert getDaysInYear(2001) == 365
   result = 365 + (if isLeapYear(year): 1 else: 0)
 
+proc getWeeksInYear*(y: int): YearweekRange =
+  ## Returns the number of weeks in a year, which can be
+  ## either 53 or 52
+  runnableExamples:
+    doAssert getWeekInYear(2000) == 52
+    doAssert getWeeksInYear(2001) == 53
```

Comment:
```suggestion
    assert getWeekInYear(2000) == 52
    assert getWeeksInYear(2001) == 53
```

https://nim-lang.github.io/Nim/contributing.html
> An exception to the above rule is runnableExamples and code-block rst blocks intended to be used as runnableExamples, which for brevity use assert instead of doAssert.